code/array/array-new-vs-fixnum-times-map.rb

```diff
@@ -0,0 +1,17 @@
+require "benchmark/ips"
+
+ELEMENTS = 9
```

Comment:
Thank you so much for the graph! I'm just getting started with this repo, so someone else should comment, but whenever there's an Array or Enumerable benchmark I'm always curious with how it scales with # of elements. You may want to add a comment explaining that this behaves pretty much the same way at 1 million elements as it does at 10.